src/OXRS_Rack32.cpp

```diff
@@ -568,6 +571,7 @@ void OXRS_Rack32::_initialiseTempSensor(void)
   
   // Set the temp sensor resolution (higher res takes longer for reading)
   _tempSensor.setResolution(MCP9808_MODE);
+  _lastTempUpdate = -_tempUpdateMs;
```

Comment:
The only issue with this is if a user has disabled temp reports via MQTT config, the config won't take effect till connection to the broker which might be after the first loop. Therefore they would end up with a temp reported on the display which would never update. 

Or perhaps we should hide the temp if we receive config to disable reports?

Reply:
async hide/unhide would require some code that blanks the line and restores the former x-pos. That's all not available now. I'll revert that line for now to get the custom LCD out.

Reply:
Sounds good